test/slack/web/documentation_test.exs

```diff
@@ -34,6 +34,41 @@ defmodule Slack.Web.DocumentationTest do
       assert {:info, 1} in module_functions
     end
 
+    test "generates function params based on required arguments from json" do
```

Comment:
ℹ️  This test is here to cover the fact that we're still properly able to generate the methods.